### ISSUE TITLE
Consider building a stand-alone BookwormDB image to be hosted on dockerhub

### ISSUE THREAD

**Issue by senderle** [senderle/bookworm-compose#13]
Right now, we're building our own image for the BookwormDB API, which is fine. But once we have everything working, we should consider whether it's worth adding a Dockerfile to the original BookwormDB repo on github, and setting up a dockerhub repo that automatically builds and tests the image based on the github repo.

The documentation explaining how to do this is here: https://docs.docker.com/docker-hub/builds/

I've done this once before — it wasn't terribly hard, and it was extremely convenient. For this project, benefits include a simpler and faster build process, and a simplified codebase. 

The only major thing to keep in mind is that this image will be publicly hosted, so we must be sure we aren't storing secrets in it, and we should probably also try to make it generic enough to be usable even by people who aren't using this compose stack.

This isn't a top priority, but I think we should return to it once we have everything working the way we want. It might be possible to do the same thing with the GUI code.

**Comment by bmschmidt** [comment]
Yeah, I think this would be useful and--coupled with a service using a pre-built mysql container--would also address some of the project's CI testing issues.
